Sources/SourceKitLSP/Swift/CursorInfo.swift

```diff
@@ -103,7 +107,10 @@ struct CursorInfo {
         isDynamic: dict[keys.isDynamic] ?? false,
         isSystem: dict[keys.isSystem] ?? false,
         receiverUsrs: dict[keys.receivers]?.compactMap { $0[keys.usr] as String? } ?? [],
-        systemModule: module
+        systemModule: module,
+        line: line,
```

Comment:
Instead of assigning to a variable, I would just use the value inline here, like we do for the other keys. Same for `column` and `length`.

```suggestion
        line: dict[keys.line] as Int?,
```